action.yml

```diff
@@ -8,9 +8,13 @@ inputs:
     description: 'Store: VAR=VALUE,OTHER_VAR=VALUE | Retrieve: VAR,OTHER_VALUE'
     required: true
   delimiter:
-    description: 'Parsing split character'
+    description: 'Regex defining the delimiter'
     required: false
-    default: "\n"
+    default: "\\r?\\n"
```

Comment:
Can you add some comment about what that regex does exactly?
I think it's a line jump, but I'm not sure. It'd help